### PR TITLE
Kick border radius of the middle button of search bar

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | There was an unwanted border radius on the middle button of the search bar when it's opened
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Go on a migrated page, open the search bar at the top and see if the dropdown button don't have any border radius on right side
| Possible impacts? | search bar

# Before
image

# After
image

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25085)

### CODE REVIEW

File: admin-dev/themes/new-theme/scss/components/layout/_main_header.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@ $header-text-color: #4e6167 !default;
 
 .main-header {
   position: fixed;
-  z-index: 1040; // modal backdrop's z-index is 1050, so this must be just below that
+  z-index: 1040; // modal backdrop"s z-index is 1050, so this must be just below that
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/scss/components/layout/_main_header.scss]: @NeOMakinG Why this change ?

File: admin-dev/themes/new-theme/scss/components/layout/_main_header.scss [hunk 2]
```diff
@@ -178,7 +184,7 @@ $header-text-color: #4e6167 !default;
           padding: 0;
           overflow: hidden;
           border: 0;
-          // we can't use display:none or else the transition doesn't work
+          // we can"t use display:none or else the transition doesn"t work
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/scss/components/layout/_main_header.scss]:
```suggestion
          // we can't use display:none or else the transition doesn't work
```

File: admin-dev/themes/new-theme/scss/components/layout/_main_header.scss [hunk 3]
```diff
@@ -264,7 +270,7 @@ $header-text-color: #4e6167 !default;
   #header-search-container {
     .dropdown-toggle {
       &::after {
-        font-size: 1.6em; // use of em so that the size is relative to the component's font size
+        font-size: 1.6em; // use of em so that the size is relative to the component"s font size
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/scss/components/layout/_main_header.scss]:
```suggestion
        font-size: 1.6em; // use of em so that the size is relative to the component's font size
```

File: admin-dev/themes/new-theme/scss/components/layout/_main_header.scss (hunk 1 above)
Comment (Progi1984) [reply admin-dev/themes/new-theme/scss/components/layout/_main_header.scss]:
```suggestion
  z-index: 1040; // modal backdrop's z-index is 1050, so this must be just below that
```

**Review by Progi1984 — COMMENTED** [review]
Some feedback

File: admin-dev/themes/new-theme/scss/components/layout/_main_header.scss (hunk 1 above)
Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/components/layout/_main_header.scss]: Ah fuck my life

**Review by PierreRambaud — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by khouloudbelguith** [comment]
Hi @NeOMakinG,

It is ok :heavy_check_mark: 
**Before**
image

**After**
image


Thank you!

**Comment by Progi1984** [comment]
Thanks @NeOMakinG ... & @khouloudbelguith of course :)
